files/image_config/hostcfgd/hostcfgd

```diff
@@ -220,6 +220,63 @@ class AaaCfg(object):
             f.write(nss_tacplus_conf)
 
 
+class k8s_config(object):
+    def __init__(self):
+        self.k8s_master = ""
+        self.insecure = False
+        self.disable = False
+
+    # Load conf from ConfigDb
+    def load(self, data):
+        if 'SERVER' in data:
```

Comment:
>if [](start = 8, length = 2)

Initial them before load?
```
        self.k8s_master = ""
        self.insecure = False
        self.disable = False
```

Reply:
load happens only once, at the start of the script. So global init is sufficient.

Reply:
I am not saying your implementation is wrong, but this is for good interface design.

Reply:
I get your point. This is the current design in hostcfgd. The load is called only from HostConfigDaemon::__init__ 
So every class initializes its member variables in its __init__; The load method sets them

Reply:
Summarizing offline discussion:
Per current design load method of each class is to be called just once only. Hence the main class calls it from its constructor only, to ensure that it is called only once. But in future, if someone may update the code, such that load method, could get called more than once, this could break the functionality.
Added a TODO comment, so every  load method to protect self from getting called more than once or handle the possibility gracefully.